src/database/migrations/20200102215044-create-accomodation.js

```diff
@@ -31,6 +31,9 @@ module.exports = {
       image: {
         type: Sequelize.STRING
       },
+      averageRate: {
```

Comment:
@mujohn26 Instead of editing a migration file, create a new migration file that alters and adds/drops the column to/from the table.